examples/anthos-cicd-with-gitlab/docs/3-set-up-anthos-config-management.md

```diff
@@ -0,0 +1,323 @@
+
+# Set up Anthos Config Management (ACM)
+
+[Anthos Config Management](https://cloud.google.com/anthos/config-management) (ACM) is a key component of Anthos that lets you define and enforce configs, including custom policies, and apply it across all your infrastructure both on-premises and in the cloud.
+
+With ACM, you can set configs and policies in one repo. Typically the security or operator team manages this repo. Using the repo model lets developers focus on app development repo(s) while the security/operators focus on infrastructure. As long as clusters are in sync with the ACM repo, changes can’t be made on policies/config managed by ACM except using the repo. This allows enterprises to take all the advantages that come with a version control system while creating and modifying configs. [Learn more](https://cloud.google.com/anthos-config-management/docs/concepts).
+
+Before setting up ACM, first enable the ACM feature:
+
+
+```
+gcloud alpha container hub config-management enable
+```
+
+
+
+## Create GitLab repo/project
+
+Click on the [GitLab $GROUP_NAME group](https://gitlab.com/dashboard/groups) you created earlier and create a new blank project called ACM. Make it public: 
+
+
+
+![Create Gitlab project](docs/images/create-blank-project.png)
+
+
+Clone the repo locally and use nomos to create the [repo structure](https://cloud.google.com/kubernetes-engine/docs/add-on/config-sync/concepts/repo) that allows [Config Sync](https://cloud.google.com/kubernetes-engine/docs/add-on/config-sync/overview) to read from the repo:
+
+
+```
+cd ~
+mkdir $GROUP_NAME
+cd $GROUP_NAME/
+git clone git@gitlab.com:$GROUP_NAME/acm.git
+cd acm/
+nomos init
+```
+
+
+To know more about the functions of the different directories created by `nomos init `you can read the [repo structure](https://cloud.google.com/kubernetes-engine/docs/add-on/config-sync/concepts/repo) documentation.
+
+
+## Deploy config sync operator to clusters
+
+The [Config Sync Operator](https://cloud.google.com/kubernetes-engine/docs/add-on/config-sync/how-to/installing#git-creds-secret) is a controller that manages Config Sync in a Kubernetes cluster.
+
+[Download](https://cloud.google.com/kubernetes-engine/docs/add-on/config-sync/downloads) the operator and deploy to your clusters:
+
+
+```
+cd ~/$GROUP_NAME/acm
+mkdir setup
+cd setup/
+gsutil cp gs://config-management-release/released/latest/config-managementsync-operator.yaml config-managementsync-operator.yaml
```

Comment:
This URL returns a 404.

There are two files there instead of one:
`gs://config-management-release/released/latest/config-management-operator.yaml `
`gs://config-management-release/released/latest/config-sync-operator.yaml `

Are these the files that have to be applied?